src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Average.txt

```diff
@@ -35,9 +35,13 @@
 2.34567
 
 //Excel returns -0.781666667
->> Average(-1.234, 2.345, -3.456)
--0.78166667
+// Rounding here so we get consistent results between float and decimal
+>> Round(Average(-1.234, 2.345, -3.456),7)
```

Comment:
>Round [](http://example.com/codeflow?start=3&length=5)

Can  BaseRunner.NumberCompare  overload let us handle this?

Reply:
Unfortunately no, decimal comparisons must be exact.  The other option is just to remove this test and rely on the one after it that returns an exact answer rather than a repeating decimal.  But I hate removing tests.  :)